### PR TITLE
add findpv(PVNAME) utility function

### LINKED ISSUE #496: What ophyd signal uses this EPICS PV?
This is a common question as an instrument package begins to grow.  Users know the EPICS PV but forgot which EPICS signal and then where is it defined.

### PR DESCRIPTION
**Pull request by prjemian**
Add `utils.findpv()` to fix #496

### CODE REVIEW

**Comment by prjemian** [comment]
@gfabbris : Ready for review

File: apstools/utils.py [hunk 1]
```diff
@@ -2325,3 +2327,203 @@ def copy_filtered_catalog(source_cat, target_cat, query=None):
...
+        Namespace dictionary of Python objects.
+
+    RETURNS
+
+    str or ``None``:
+        Name of the ophyd object.
+
+    EXAMPLE::
+
+        In [45]: findpv("adsimdet_cam_acquire")
```
Comment (gfabbris) [inline apstools/utils.py:2488]: typo? should it be `findname`?

**Review by gfabbris — APPROVED** [review]
Looks great!!

File: apstools/utils.py (hunk 1 above)
Comment (prjemian) [reply apstools/utils.py:2488]: yup, that's a copy/paste error caught in its native habitat

**Pull request merged**